core/app/models/spree/product_property.rb

```diff
@@ -9,7 +9,7 @@ class ProductProperty < Spree::Base
 
     validates :property, presence: true
 
-    validates :value, db_maximum_length: true
+    validates :value, db_maximum_length: true, presence: true
```

Comment:
@tanmay3011 DbMaximumLengthValidator adds maximum length validation error only if the value of attribute is present. So there won't be any case when both validations fail together.

```
class DbMaximumLengthValidator < ActiveModel::EachValidator
  def validate_each(record, attribute, value)
    limit = record.class.columns_hash[attribute.to_s].limit
    value = record[attribute.to_sym]
    if value && limit && value.to_s.length > limit
      record.errors.add(attribute.to_sym, :too_long, count: limit)
    end
  end
end
```